src/vma/sock/sockinfo_tcp.h

```diff
@@ -293,6 +293,7 @@ class sockinfo_tcp : public sockinfo, public timer_handler
 	ready_pcb_map_t m_ready_pcbs;
 	static const unsigned TX_CONSECUTIVE_EAGAIN_THREASHOLD = 10;
 	unsigned m_tx_consecutive_eagain_count;
+	vVMA_spec_t 	m_mce_spec;
```

Comment:
vVMA_spec_t is not new in this commit. However looking in sys_var.h the naming convention would require this type to be named:
```
vma_spec_t
```